ooniprobe/View/Informed consent/Onboarding4ViewController.m

```diff
@@ -0,0 +1,89 @@
+#import "Onboarding4ViewController.h"
+
+@interface Onboarding4ViewController ()
+
+@end
+
+@implementation Onboarding4ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    //Constraint for iPhoneSE
+    if(UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone)
```

Comment:
Also, maybe, if this is a stack overflow answer, can you add comment linking to it?

Reply:
This is my code

Reply:
ok!